### ISSUE TITLE
Support event statistics view in Theia

### ISSUE THREAD

**Issue by tahini** [theia-ide/theia-trace-extension#16]
This is the type of categorical view that can be used as base output

- [ ] Make sure the data is provided through a data provider in Trace Compass
- [ ] Support this data type with pie chart in Theia

![eventStatistics](https://user-images.githubusercontent.com/4398897/82827366-e4347d80-9e7c-11ea-8abb-082e96d25f8a.png)

**Comment by MatthewKhouzam** [comment]
Pie charts remain, I would suggest they are blocked by D3's progress. @Rodrigoplp-work thoughts?

**Comment by Rodrigoplp-work** [comment]
Exactly! With the structure we are already deploying in the y-axis plus the examples discussed in the "0003-D3-charts" ADR, this would fall perfectly in line for expanding our use of the library.
